ext/xml/xml.stub.php

```diff
@@ -146,35 +146,23 @@ function xml_parser_create_ns(?string $encoding = null, string $separator = ":")
 
 function xml_set_object(XMLParser $parser, object $object): true {}
 
-/**
- * @param callable $start_handler
- * @param callable $end_handler
- */
-function xml_set_element_handler(XMLParser $parser, $start_handler, $end_handler): true {}
```

Comment:
Hi @Girgias you changed the stub signature from callable to callable|string|null.

Can you confirm this is not a PHP 8.4-related change, and this should have been the original param phpdoc ?